src/main/java/it/pagopa/transactions/commands/handlers/v2/TransactionRequestAuthorizationHandler.java

```diff
@@ -95,7 +95,7 @@ public TransactionRequestAuthorizationHandler(
                 "walletAsyncQueueClient"
             ) Optional<WalletAsyncQueueClient> walletAsyncQueueClient,
             @Value("${azurestorage.queues.transientQueues.ttlSeconds}") Integer transientQueuesTTLSeconds,
-            @Value("${npg.authorization.request.timeout.seconds}") Integer npgAuthRequestTimeout,
+            @Value("${authorization.event.visibilityTimeoutSeconds}") Integer saveLastUsedMethodTimeout,
```

Comment:
If the first step of event AuthRequest changes in the event-dispatcher this variable name (_saveLastUsedMethodTimeout_) may give incorrect information. 
Might it make sense to leave a more generic name for the timeout? (e.g. _AuthRequestEventTimout_)

Reply:
yes, miss this during refactoring made [here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/583/commits/dfe461467bfa0490972a9137341cf1a9bf67a38e) 😄 fixing it, good catch

Reply:
fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/583/commits/da90b681b3e4db2f6d9357c18c9ce25692e3c9d1